index.html

```diff
@@ -3098,6 +3098,119 @@ <h1>Speakers</h1>
+                                            <a target="default" href="https://www.facebook.com/angularboy"><i class="icon social_facebook"></i></a>
+                                            <a target="default" href="https://www.linkedin.com/in/angularboy/"><i class="icon social_linkedin"></i></a>
+                                            <a target="default" href="https://github.com/aayusharora"><i class="icon fa fa-github"></i></a>
+                                        </div>
+                                    </div>
+                              </div>
+                              <span class="speaker-name">Aayush Arora</span>
+                              <span>Web Developer</span>
+                              <span>FOSSASIA</span>
+                            </div>
+                        </div>
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                              <div class="image-holder">
+                                    <img class="background-image" alt="Pronnoy Goswami" src="img/PronnoyGoswami.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="https://www.linkedin.com/in/pronnoygoswami/"><i class="icon fa fa-external-link"></i></a>
+                                            <a target="default" href="https://twitter.com/pronnoy_goswami"><i class="icon social_twitter"></i></a>
+                                            <a target="default" href="https://www.linkedin.com/in/pronnoygoswami/"><i class="icon social_linkedin"></i></a>
+                                            <a target="default" href="https://github.com/goswamipronnoy"><i class="icon fa fa-github"></i></a>
+                                        </div>
+                                    </div>
+                              </div>
+                              <span class="speaker-name">Pronnoy Goswami</span>
+                              <span>Software Development Engineer</span>
+                              <span>McKinsey & Company</span>
+                            </div>
+                        </div>
+
+                        <div class="col-md-3 col-sm-6 speaker-column">
+                            <div class="speaker">
+                              <div class="image-holder">
+                                    <img class="background-image" alt="Himanshu Singh" src="img/HimanshuSingh.jpg">
+                                    <div class="hover-state text-center preserve3d">
+                                        <div class="social-links vertical-align">
+                                            <a target="default" href="https://www.linkedin.com/in/himanshusingh2407"><i class="icon fa fa-external-link"></i></a>
```

Comment:
External link of Himanshu Singh is given, but you have added likedIn link of him. Check at eventyay.

Reply:
@simsausaurabh Himanshu Singh's external link contains LinkedIn profile only at eventyay, so I've removed the External Link Button for him as well. Please review.